runners/java-fn-execution/src/test/java/org/apache/beam/runners/fnexecution/control/RemoteExecutionTest.java

```diff
@@ -584,21 +584,28 @@ public void onProgress(ProcessBundleProgressResponse progress) {}
           @Override
           public void onCompleted(ProcessBundleResponse response) {
             // Assert the timestamps are non empty then 0 them out before comparing.
-            List<MonitoringInfo> actualMIs = new ArrayList<>();
+            List<MonitoringInfo> actual = new ArrayList<MonitoringInfo>();
             for (MonitoringInfo mi : response.getMonitoringInfosList()) {
               MonitoringInfo.Builder builder = MonitoringInfo.newBuilder();
               Assert.assertTrue(mi.getTimestamp().getSeconds() > 0);
               builder.mergeFrom(mi);
               builder.clearTimestamp();
-              actualMIs.add(builder.build());
+              actual.add(builder.build());
             }
 
             SimpleMonitoringInfoBuilder builder = new SimpleMonitoringInfoBuilder();
             builder.setUrnForUserMetric(RemoteExecutionTest.class.getName(), counterMetricName);
             builder.setInt64Value(2);
-            MonitoringInfo expectedCounter = builder.build();
+            MonitoringInfo userCounter = builder.build();
 
-            assertThat(actualMIs, CoreMatchers.hasItems(expectedCounter));
+            builder = new SimpleMonitoringInfoBuilder();
+            builder.setUrn(SimpleMonitoringInfoBuilder.ELEMENT_COUNT_URN);
+            builder.setPCollectionLabel("impulse.out");
+            builder.setInt64Value(2);
```

Comment:
Use unique values. Right now you have 3 places that use same values: element_count counter, userMetric counter and total amount of counters.
This might lead to misunderstanding.

Reply:
Done